src/Microsoft.VisualStudio.ProjectSystem.IntegrationTests/Utilities/FeatureAttribute.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.IntegrationTests
+{
+    internal partial class FeatureAttribute
+    {
+        public const string AutomaticEndConstructCorrection = nameof(AutomaticEndConstructCorrection);
```

Comment:
These are used to wait for features.  They should be moved into the test framework itself so that things like `VisualStudio.Workspace.WaitForAsyncOperations(FeatureAttribute.SolutionCrawler)` work.  Arguable everything in the Utilities folder should be moved to the test framework itself.  I'll talk to @tannergooding about it.